pandas/core/frame.py

```diff
@@ -4401,6 +4402,15 @@ def pivot(self, index=None, columns=None, values=None):
         one  1   2   3
         two  4   5   6
 
+        >>> df.pivot(index='foo', columns='bar', values=['baz'])
```

Comment:
oh we already have this example, I c, ok you can remove this one (with ``['bar']``)